test/uk/gov/hmrc/pillar2/service/FinancialServiceSpec.scala

```diff
@@ -34,37 +34,88 @@ class FinancialServiceSpec extends BaseSpec with Generators with ScalaCheckPrope
 
   private val service = new FinancialService(mockFinancialDataConnector)
 
+  val startDate = LocalDate.now()
+  val endDate   = LocalDate.now().plusDays(364)
```

Comment:
What about leap years ?
Would it be better to have:
val endDate = LocalDate.now().plusYear(1).minusDays(1)